src/audio/pipeline.c

```diff
@@ -312,10 +312,10 @@ int pipeline_complete(struct pipeline *p)
 }
 
 /* connect component -> buffer */
-int pipeline_comp_connect(struct pipeline *p, struct comp_dev *source_comp,
-	struct comp_buffer *sink_buffer)
+int pipeline_comp_connect(struct comp_dev *source_comp,
+			  struct comp_buffer *sink_buffer)
 {
-	trace_pipe_with_ids(p, "pipeline_comp_connect()");
+	trace_pipe("pipeline_comp_connect()");
```

Comment:
Can you take a look at handler.c as I've made some update there to make trace more parsible and with more context. e.g this message would become "pipe: %d connect  source %d -> sink %d", pipeline_id, source_id, sink_id);  Can you do this to all your messages.

Reply:
@lgirdwood 
I've updated this traces, but still I am not able to print pipeline id from the same reason, which I wrote above: https://github.com/thesofproject/sof/pull/578#issuecomment-438631482. 

The only way to identify pipeline here is to use for example: source_buffer-> ipc_buffer.comp.pipeline_id, but it refers to component_id in struct sof_ipc_pipe_new (not pipeline_id).